src/styles/layouts/_layout-product-detail.scss

```diff
@@ -0,0 +1,45 @@
+.layout-product-detail {
```

Comment:
`layout-product-view`

Reply:
Which name should we use the name of `page` or `component`?